src/containers/AddPlant/AddPlant.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import Input from '../../components/Input';
+import Select from '../../components/Select';
+import styles from './AddPlant.module.scss';
+import PlantPic from '../../assets/illustrations/plant-leaf.png';
+import RemovePic from '../../assets/icons/Paper Fail.png';
+import Button from '../../components/Button/Button';
+import Text from '../../components/Text/Text';
+import Modal from '../Modal/Modal';
+
+const AddPlant = () => {
+  const [show, setShow] = useState(false);
+  const openModal = () => setShow(true);
+  const closeModal = () => setShow(false);
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.basicInfo}>
+        <div className={styles.title}>
+          <Text text="Help us grow!" fontsize="2em" />
+          <Text text="If you did not find your plant in our base, you can add it below." fontsize="2em" />
+        </div>
+        <Modal closeModal={closeModal} show={show} />
+        <Input className={styles.input} text="Name" />
+        <Input className={styles.input} text="Latin Name" />
+        <Select className={styles.select} title="Species" />
+      </div>
+      <div className={styles.detailedInfo}>
+        <Select className={styles.select} title="Min temperature" values={[10, 15, 20]} />
```

Comment:
Why do you enter class names for imported components? It doesn't make sense, if you want to style them appeal through children.